users/tests.py

```diff
@@ -819,6 +935,106 @@ def test_register_assessor_with_invalid_one_time_code(self):
         response_content = json.loads(response.content)
         self.assertEqual(response_content['message'], expected_message)
 
+    
+
+class AssesseeViewsTestCase(TestCase):
+        def setUp(self) -> None:
+            self.registration_base_data = {
+                    'email': 'assessee@gmail.com',
+                    'password': 'testPassword1234',
+                    'confirmed_password': 'testPassword1234',
+                    'first_name': 'Anastasia',
+                    'last_name': 'Yuliana',
+                    'phone_number': '+6281275725231',
+                    'date_of_birth': '1994-09-30T10:37:35.849Z'
+                }
+
+        def fetch_with_data(self, registration_data, url_to_fetch):
+            response = self.client.post(
+                url_to_fetch,
+                data=json.dumps(registration_data),
+                content_type='application/json'
+            )
+            return response
+
+        def test_register_assessee_when_complete(self):
+            registration_data = self.registration_base_data.copy()
+            response = self.fetch_with_data(registration_data, REGISTER_ASSESSEE_URL)
+            self.assertEqual(response.status_code, OK_REQUEST_STATUS_CODE)
```

Comment:
Hello, Arya! After the last commit, we are moving to use HTTPStatus instead of our own defined constant. Can you please change it in your following commits? Thanks!